dmoj/packet.py

```diff
@@ -209,7 +212,7 @@ def _receive_packet(self, packet):
         else:
             log.error('Unknown packet %s, payload %s', name, packet)
 
-    def handshake(self, problems, runtimes, id, key):
+    def handshake(self, problems: str, runtimes: str, id: str, key: str):
```

Comment:
`runtimes` is not `str`, it's some really complex object (triply-nested tuples?) that might be better to leave untyped for now.